hack/e2e.sh

```diff
@@ -293,12 +298,10 @@ function test_cassandracluster() {
 
     # Change the CQL port
     export CASS_CQL_PORT=9043
-    kubectl apply \
-        --namespace "${namespace}" \
-        --filename \
-        <(envsubst \
-              '$NAVIGATOR_IMAGE_REPOSITORY:$NAVIGATOR_IMAGE_TAG:$NAVIGATOR_IMAGE_PULLPOLICY:$CASS_NAME:$CASS_REPLICAS:$CASS_CQL_PORT:$CASS_VERSION' \
-              < "${SCRIPT_DIR}/testdata/cass-cluster-test.template.yaml")
+    if ! apply_cassandracluster
+    then
+        fail_test "Failed to apply cassandracluster"
+    fi
```

Comment:
:grey_question: personally, I wouldn't bother with the `fail_test` part here. I don't regard "applying the desired configuration" as an E2E test....just an implementation detail of the test....so I wouldn't bother adding a specific test error message.
Also, if this command fails, then all bets are off and we should probably just exit the test so that it's easy to debug the problem.
But if you prefer to leave as-is, until we land the gingko test branch, I'm happy with that.